lib/lotus/helpers/numbers.rb

```diff
@@ -0,0 +1,59 @@
+require 'lotus/helpers/numbers/number_formatter'
+
+module Lotus
+  module Helpers
+    # Number formatter
+    #
+    # You can include this module inside your view and 
+    # the view will have access all methods.
+    #
+    # By including <tt>Lotus::Helpers::Numbers</tt> it will inject private
+    # methods as markup escape utilities.
+    #
+    # @since x.x.x
+    module Numbers
+      private
+      # Number formatting
+      #
+      # This helper formats raw numbers to make them more human readable
+      #
+      # @param number [Numeric] the number to be formatted
+      #
+      # @return [String] formatted number
+      #
+      # @since x.x.x
+      #
+      # @see Lotus::Helpers::Numbers#format_number
+      #
+      # @example Basic usage
+      #   require 'lotus/helpers/numbers'
+      #
+      #   class checkout
+      #     include Lotus::Helpers::Numbers
+      #
+      #     def total
+      #       n 1999.99
+      #     end
+      #
+      #     def euros
+      #       n 1256.95, delimiter: '.', separator: ','
+      #     end
+      #   end
+      #
+      #   view = MyView.new
+      #
+      #   view.total
+      #     # => "1,999.99"
+      #
+      #   view.euros
+      #     # => "1.256,95"
+      def format_number(number, options = {})
+        number_parser = NumberFormatter.new(number, options)
+        number_parser.parse
+      end
+
+      # @since x.x.x
+      alias_method :n, :format_number
```

Comment:
We have defined `#h` et co because it's a well known idea in Ruby web frameworks, and because we want to have less noise in HTML generator.

But we should limit as much as possible to overload views contexts. What do you think if we remove it for now?